cloudfront/wellcomelibrary.org/edge-lambda/src/wellcomeLibraryRedirect.test.ts

```diff
@@ -61,6 +69,57 @@ const rewriteTests = (): ExpectedRewrite[] => {
       in: '/events/any-thing',
       out: expectedRedirect('https://wellcomecollection.org/whats-on'),
     },
+    // API uris redirect
+    {
+      in: '/iiif/collection/b18031900',
+      out: expectedCORSRedirect('https://iiif.wellcomecollection.org/presentation/v2/b18031900'),
+      data: 'https://iiif.wellcomecollection.org/presentation/v2/b18031900',
+    },
+    {
+      in: '/iiif/collection/b18031900',
+      out: expectedServerError('Got 404 from https://iiif.wellcomecollection.org/wlorgp/iiif/collection/b18031900'),
+      error: axios404
+    },
+    {
+      in: '/iiif/collection/b18031900',
+      out: expectedServerError('No response from https://iiif.wellcomecollection.org/wlorgp/iiif/collection/b18031900'),
+      error: axiosNoResponse
+    },
```

Comment:
Very minor: it wasn't immediately clear to me where these URLs came from, and whether these test cases had been fleshed out properly. Picking separate b numbers for the tests might have made that clearer (and be easier to debug in tests.)